src/main/java/com/vacation_bot/domain/models/VacationTotal.java

```diff
@@ -13,7 +13,7 @@
 @EqualsAndHashCode
 @Document
 @CompoundIndex(name = "unique_idx", def = "{ 'user-id': 1, 'year': 1 }", unique = true)
-public class VacationTotalModel {
+public class VacationTotal {
```

Comment:
Minor note: it's better to use the project convention and name class as `VacationTotalModel` and collection name as `@Document( collection = "vacationTotal" )`. It's not necessary to change I just want to note that the project style should be the same for all classes. It simplifies a maintenance.